README.md

```diff
@@ -12,21 +12,25 @@ This macro is intended to be used within a salesforce dbt project model. To leve
 ```yml
 packages:
   - package: fivetran/salesforce_formula_utils
-    version: [">=0.3.0", "<0.4.0"]
+    version: [">=0.4.0", "<0.5.0"]
 ```
 > **Note**: In order to use the macros included in this package you will need to have a properly configured source package with a source named `salesforce`. To see an example of a properly configured Salesforce source yml you can reference [integration_tests](integration_tests/models/src_fivetran_formula.yml). You are also welcome to copy/paste this source configuration into your dbt root project and modify for your Salesforce use case.
 
-### Model Creation
-Once the package is added, you may use the macro within your salesforce models. To do so you will create a new file in your models folder and name it (`your_table_name_here`.sql). Then add the below snippet into the file. You will then update the `your_table_name_here` argument to be the table for which you are generating the model:
+### Model Creation: All relevant formula fields
```

Comment:
wondering if we can make it more apparent that this section and the following one are alternative and mutually exclusive options (or, there is a good deal of overlap tbh, like they only differ in talking about the `fields_to_include` argument, so perhaps some of this info can be consolidated to make what differentiates them clearer/more salient)

Reply:
Yeah the inclusion of the `fields_to_include` argument completely changes the functionality of the macro, so I wanted to make them be different methods. Let's chat about how we can make this more salient.

Reply:
clean this up in the readme having them be two sub sections being mutual exclusive

Reply:
Update for this will be present in the next commit. Will want to confirm approach before release.